test/lost-masonry-column.js

```diff
@@ -3,36 +3,70 @@
 var check = require('./check');
 
 describe('lost-masonry-column', function() {
-  it('supports a custom gutter', function() {
-    check(
-      'a { lost-masonry-column: 60px; }',
-      'a { float: left; width: calc(99.9% * 60px - 30px); margin-left: 15px;' +
-      ' margin-right: 15px; }'
-    );
-  });
+  describe('Custom Gutter', function() {
+    it('supports a custom gutter', function() {
+      check(
+        'a { lost-masonry-column: 60px; lost-masonry-column-gutter: 20px; }',
+        'a { float: left; width: calc(99.9% * 60px - 20px); margin-left: 10px;' +
+        ' margin-right: 10px; }'
+      );
+    });
+
+    it('supports no gutter', function() {
+      check(
+        'a { lost-masonry-column: 60px; lost-masonry-column-gutter: 0px; }',
+        'a { float: left; width: calc(99.9% * 60px - 0px); margin-left: 0;' +
+        ' margin-right: 0; }'
+      );
+    });
 
-  it('supports no gutter', function() {
-    check(
-      'a { lost-masonry-column: 0; }',
-      'a { float: left; width: calc(99.9% * 0 - 30px); margin-left: 15px;' +
-      ' margin-right: 15px; }'
-    );
+    it('supports a null unit for "no gutter"', function() {
```

Comment:
I'm not sure you need a separate test here. I think you could add the `check` done here to the test above `supports no gutter`. You are essentially checking the same thing, just whether they have added the unit to the end or not. Only very minor thing, so by no means essential.